packages/styles/src/typography/body.css

```diff
@@ -0,0 +1,20 @@
+/**
+ * Typographic text classes
+ * @variant {x-small | small | medium | large} syn-body The font size of the body text
+ * @variant syn-display-inverted Show the display text in inverted color
```

Comment:
I think we have a leftover from the boolean css variable testing here. 
We should remove it

```suggestion
```